modules/my_health/spec/request/v1/threads_request_spec.rb

```diff
@@ -12,100 +12,146 @@
-        end
+          it 'responds to GET #index when camel-inflected' do
+            VCR.use_cassette('sm_client/threads/gets_threads_in_a_folder_camel') do
+              get "/my_health/v1/messaging/folders/#{inbox_id}/threads",
+                  params: { page_size: '5', page_number: '1', sort_field: 'SENDER_NAME', sort_order: 'ASC' },
+                  headers: { 'X-Key-Inflection' => 'camel' }
+            end
 
-        it 'responds to GET #index when camel-inflected' do
-          VCR.use_cassette('sm_client/threads/gets_threads_in_a_folder_camel') do
-            get "/my_health/v1/messaging/folders/#{inbox_id}/threads",
-                params: { page_size: '5', page_number: '1', sort_field: 'SENDER_NAME', sort_order: 'ASC' },
-                headers: { 'X-Key-Inflection' => 'camel' }
+            expect(response).to be_successful
+            expect(response.body).to be_a(String)
+            expect(response).to match_camelized_response_schema('message_threads')
           end
 
-          expect(response).to be_successful
-          expect(response.body).to be_a(String)
-          expect(response).to match_camelized_response_schema('message_threads')
+          it 'returns an empty array when there are no messages in the folder' do
+            VCR.use_cassette('sm_client/threads/gets_threads_in_a_folder_no_messages') do
+              get "/my_health/v1/messaging/folders/#{inbox_id}/threads",
+                  params: { page_size: '5', page_number: '1', sort_field: 'SENDER_NAME', sort_order: 'ASC' }
+            end
+
+            expect(response).to be_successful
+
+            json_response = JSON.parse(response.body)
+            expect(json_response).to eq({ 'data' => [] })
+            expect(response).to match_response_schema('message_threads_no_messages')
+          end
         end
+      end
 
-        it 'returns an empty array when there are no messages in the folder' do
-          VCR.use_cassette('sm_client/threads/gets_threads_in_a_folder_no_messages') do
-            get "/my_health/v1/messaging/folders/#{inbox_id}/threads",
-                params: { page_size: '5', page_number: '1', sort_field: 'SENDER_NAME', sort_order: 'ASC' }
+      describe '#move' do
+        let(:thread_id) { 7_065_799 }
+
+        it 'responds to PATCH threads/move' do
+          VCR.use_cassette('sm_client/threads/moves_a_thread_with_id') do
+            patch "/my_health/v1/messaging/threads/#{thread_id}/move?folder_id=0"
           end
 
           expect(response).to be_successful
-
-          json_response = JSON.parse(response.body)
-          expect(json_response).to eq({ 'data' => [] })
-          expect(response).to match_response_schema('message_threads_no_messages')
+          expect(response).to have_http_status(:no_content)
         end
       end
     end
+  end
+
+  context 'when legacy sm policy' do
+    before do
+      Flipper.disable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs